data/brands/shop/convenience.json

```diff
@@ -5798,6 +5798,18 @@
         "name:zh": "超市发",
         "shop": "convenience"
       }
+    },
+    {
+      "displayName": "十足",
+      "id": "shizu-zhejiang-cn",
+      "locationSet": {"include": ["cn"]},
+      "tags": {
+        "brand": "十足",
+        "brand:en": "Shizu",
```

Comment:
https://wiki.openstreetmap.org/wiki/Multilingual_names#Repeating_name_with_language_specific_tag

It is recommend to repeat `brand:zh` as same as `brand`

Reply:
Updated